HopsanGUI/GraphicsView.cpp

```diff
@@ -141,6 +141,46 @@ void GraphicsView::contextMenuEvent ( QContextMenuEvent * event )
     mIgnoreNextContextMenuEvent = false;
 }
 
+void GraphicsView::mouseDoubleClickEvent(QMouseEvent *event)
```

Comment:
This breaks entering subsystems by double-click

But it solves the problem of the box always popping up, I assume that some forwarding of the event, or calling parent double-click event, (I dont remember how this was done before), is needed.

Reply:
Actually it breaks all double-clicking on things, like scopes or components for properties